dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -60,6 +60,12 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
     winrt::Microsoft::Windows::AppNotifications::AppNotificationManager AppNotificationManager::Default()
     {
         auto appProperties{ winrt::CoreApplication::Properties() };
+        static std::once_flag flag;
+        std::call_once(flag, []()
```

Comment:
Looking at the code below, you don't even need this call_once lambda. You can do this in the else clause below which is under the lock.

Reply:
Fixed.